config/dev.webpack.config.js

```diff
@@ -34,9 +34,9 @@ plugins.push(
       root: resolve(__dirname, '../'),
       exposes: {
         './RootApp': resolve(__dirname, '../src/AppEntry'),
-        './SystemDetail': resolve(
+        './SystemAdvisor': resolve(
```

Comment:
I wouldn't rename the exported module name, we can keep the `./SystemDetail`. After all it's a system's detail, isn't it? You can keep the `src/SmartComponents/SystemAdvisor` file name though.

Reply:
ah! gottcha on the logic why, onboard with that, will change it back